data/categories.lua

```diff
@@ -102,11 +102,21 @@ function categories:SetCategoryState(kind, category, enabled)
   database:SetItemCategoryEnabled(kind, category, enabled)
 end
 
+---@param category string The name of the custom category to create.
 function categories:CreateCategory(category)
   database:CreateCategory(category)
   events:SendMessage('categories/Changed')
 end
 
+---@param category string The name of the custom category to rename.
+---@param newName string The new name of the category.
+function categories:RenameCategory(category, newName)
+  database:RenameCategory(category, newName)
```

Comment:
Add `categories:DeleteCategory(category)` after this line to remove the old category otherwise you get left with the old category with no items associated with it